<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="jeffrey-pt7279" failures="0" tests="4" name="operationsTest.customerExceptionTest" time="0.326" errors="0" timestamp="2024-05-09T15:37:22 IST" skipped="0">
  <testcase classname="operationsTest.customerExceptionTest" name="getAccountDetails" time="0.005"/>
  <system-out/>
  <testcase classname="operationsTest.customerExceptionTest" name="getAccountDetails" time="0.001"/>
  <system-out/>
  <testcase classname="operationsTest.customerExceptionTest" name="getAccountDetails" time="0.020"/>
  <system-out/>
  <testcase classname="operationsTest.customerExceptionTest" name="getAccountDetails" time="0.300"/>
  <system-out/>
</testsuite> <!-- operationsTest.customerExceptionTest -->
